docs/source/tutorials/day_of_the_week.qmd

```diff
@@ -280,34 +322,99 @@ As a result, we can see the posterior distribution of our novel day-of-the-week
 ```{python}
 # | label: fig-output-day-of-week
 # | fig-cap: Day of the week effect
-out = hosp_model_dow.plot_posterior(
-    var="dayofweek_effect_raw", ylab="Day of the Week Effect", samples=500
+# Create an InferenceData object from hosp_model_dow
+dow_idata = az.from_numpyro(
+    posterior=hosp_model_dow.mcmc,
 )
 
-sp = hosp_model_dow.spread_draws(["dayofweek_effect_raw"])
-# dayofweek_effect is not recorded
+# Extract day of week effect (DOW)
+dow_effect_raw = dow_idata.posterior["dayofweek_effect_raw"]
+dow_effect_raw = dow_effect_raw.squeeze()
+dow_effect_raw = dow_effect_raw.T
+indices = np.random.choice(dow_effect_raw.shape[1], size=200, replace=False)
+dow_plot_samples = dow_effect_raw[:, indices]
+fig, ax = plt.subplots(1, 1, figsize=(8, 6))
```

Comment:
Is there a reason this isn't done with `plot_ts` like the other plots?

Reply:
Yes, looking over [this link](https://python.arviz.org/en/latest/api/generated/arviz.plot_ts.html) is a semi-prerequisite for the below. 

The `y` parameter operates from variables in the xarray dataset `observed_data`, which is standard to `InferenceData` objects. All instances of creating `idata` using `az.from_numpyro()` have had the variables `negbinom_rv` and its dimensions in the the `observed_data` dataset. 

Some data variables we want from the posterior (e.g. `Rt` or `dayofweek_effect_raw`) are not in the `observed_data` dataset, but rather in the xarray dataset `idata.posterior`. 

How then may be connect `idata.posterior["variable_we_want_to_plot"]` to `az.plot_ts(...)`? The method I opted for was to use `idata.assign(observed_data=<xarray_dataset_for_the_posterior>` (using `plot_ts()` in a way I presume was not intended). This leads to further issues, since the dimensions of the `observed_data` dataset will include the original dimensions. An example: Dimensions:  (negbinom_rv_dim_0: 90, chain: 1, draw: 2000, dayofweek_effect_raw_dim_0: 7). This "multidimensional" `y` requires then the use of `plot_dim` and `x`, which are usually manual. 

I opt for brevity to not use `plot_ts()` in instances like this.